source/_posts/2017-03-30-Haptic-feedback-makes-you-vibrate.md

```diff
@@ -0,0 +1,147 @@
+---
+title:  Haptic Feedback makes you vibrate!
+date: 2017-03-30 10:10:57
+tags: ios,haptic,feedback,vibrate
+authorIds: 
+- maco
+categories:
+- iOS
+---
+
+The iPhones could vibrate ever since they were launched. And so did almost all the other phones, even from before smartphones were a thing. But Apple took this function further than anyone else, by introducing the Taptic Engine in the Apple Watch, and after on in many other products, like the new MacBooks and iPhones. The Taptic Engine provides a haptic feedback, more similar to real-life touches. 
```

Comment:
more similar to real-life touches than standard vibration. (Something along those lines. If you use "more" then you're kind of comparing it to something else)